src/high_level.rs

```diff
@@ -130,6 +130,24 @@ pub fn load_header(index: usize, source: Source) -> Result<Header, SysError> {
     }
 }
 
+/// Load witness
+///
+/// Return the witness or a syscall error
+///
+/// # Arguments
```

Comment:
The Arguments and Example sections are not providing any actual value. They are just retierating the function signature.

Reply:
I just follow the current document style, for example: https://github.com/nervosnetwork/ckb-std/blob/fa3823a83e795079c277843307dc68eff8aacb2b/src/high_level.rs#L133-L146

I agree that we should improve the document quality, let's do it in a separate PR.